r2/r2/controllers/api.py

```diff
@@ -870,6 +931,11 @@ def POST_unfriend(self, nuser, iuser, container, type):
         if type in ("friend", "enemy") and container != c.user:
             abort(403, 'forbidden')
         fn = getattr(container, 'remove_' + type)
+
+        # if the top mod is removing themself
```

Comment:
This comment is slightly inaccurate: should be "if the top mod is removing themself or an admin is removing them"